tutorpicasso/commands/run_extra_commands.py

```diff
@@ -0,0 +1,142 @@
+import re
+import subprocess
+
+# Was necessary to use this for compatibility with Python 3.8
+from typing import Any, List
+
+import click
+from tutor import config as tutor_config
+
+COMMAND_CHAINING_OPERATORS = ["&&", "&", "||", "|", ";"]
+
+
+@click.command(name="run-extra-commands", help="Run tutor commands")
+def run_extra_commands() -> None:
+    """
+    This command runs tutor commands defined in PICASSO_EXTRA_COMMANDS
+    """
+    tutor_root = (
+        subprocess.check_output("tutor config printroot", shell=True)
+        .decode("utf-8")
+        .strip()
+    )
+    config: Any = tutor_config.load(tutor_root)
+    picasso_extra_commands: Any = config.get("PICASSO_EXTRA_COMMANDS", None)
+    if picasso_extra_commands is not None:
+        validate_commands(picasso_extra_commands)
+        for command in picasso_extra_commands:
+            run_command(command)
+
+
+def validate_commands(commands: Any) -> None:
```

Comment:
Consider using this helper function only to check whether a command is valid and return the result. This way, the command is responsible for raising click exceptions instead of raising them in the helper function.